e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -159,78 +159,65 @@ test.describe('Example Imagery Object', () => {
     });
 
     test('Can use + - buttons to zoom on the image', async ({ page }) => {
-        await page.locator(backgroundImageSelector).hover({trial: true});
-        const zoomInBtn = page.locator('.t-btn-zoom-in').nth(0);
-        const zoomOutBtn = page.locator('.t-btn-zoom-out').nth(0);
+        // Get initial image dimensions
         const initialBoundingBox = await page.locator(backgroundImageSelector).boundingBox();
 
-        await zoomInBtn.click();
-        await zoomInBtn.click();
-        // wait for zoom animation to finish
-        await page.locator(backgroundImageSelector).hover({trial: true});
+        // Zoom in twice via button
+        await buttonZoomIn(page);
```

Comment:
Let's rename this function while we're at it

"ImageryZoomInAction"
"Zoom into imagery"

Something along those lines